.env.new

```
@@ -0,0 +1,6 @@
+# Database Configuration
+POSTGRES_URI=postgresql://postgres:postgres123@localhost:5432/vanta_ledger
+MONGO_URI=mongodb://localhost:27017/vanta_ledger
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MONGO_URI key should go before the POSTGRES_URI key

(UnorderedKey)